src/en/clean-copy/01-Preface/01.md

```diff
@@ -1,13 +1,13 @@
 ### Author's Note
 
-The history of beer and brewing was always a practical study for me. I was always keen to try something new — and not just try, but learn from it: when this beverage occurred and why it tastes like this. Gradually, I involved myself in studying beer history and soon found that brewing was one of the hottest topics of historical science.
+The history of beer and brewing was always a practical study for me. I was always keen to try something new, not just to try but also learn from it. How was this beverage produced and why does it taste like this? Gradually, I became interested in studying beer history and soon found that brewing was one of the hottest, even intriguing, topics of historical dialogue.
 
-For many centuries in northern parts of Europe beer was if not the first sector of economics then at least second — the fact that both Medieval chroniclers and contemporary scholars turn a blind eye to. Up until the 1980s, the researchers were interested in the history of alcohol only in the context of consumption effects on personal and public health[ref:{"source":{"alias":"dietler-alcohol","short":"Dietler, M. (2006)","extra":\["Alcohol: Anthropological/Archaeological Perspectives","Annual Review of Anthropology, 2006. 35:229–49:230"\],"href":"https://www.academia.edu/272118/Alcohol_Anthropological_Archaeological_Perspectives"}}:230](). But as we all well know, alcohol, this ‘social lubricant’, plays a much greater role in society! Gradually, this fact reached academic studies. In the case of beer, it happened even later, at the beginning of the 21st century. It turns out that beer, an ordinary and democratic beverage, allows us to poke our noses into the most interesting and least documented part of the past: the daily routine of common folk.
+For many centuries in parts of Northern Europe, beer was if not the primary economic driver then it was at least second — a fact that both Medieval chroniclers and contemporary scholars turn a blind eye to. Up until the 1980's, researchers were largely interested in the history of alcohol only in the context of consumption effects on personal and public health[ref:{"source":{"alias":"dietler-alcohol","short":"Dietler, M. (2006)","extra":\["Alcohol: Anthropological/Archaeological Perspectives","Annual Review of Anthropology, 2006. 35:229–49:230"\],"href":"https://www.academia.edu/272118/Alcohol_Anthropological_Archaeological_Perspectives"}}:230](). Yet as we all well know, alcohol, this ‘social lubricant’, plays a much greater role in society. Gradually, this fact reached academic circles. In the case of beer, it happened even later at the turn of the 21st century. It turns out that beer, an ordinary and democratic beverage, allows us to poke our noses into the most interesting and least documented part of the past; the daily experiences and occupations of common folk.
```

Comment:
"turn of the 21st century. It turns", duplicate 'turn'. Maybe, return 'beginning'?

Reply:
Is there a way I can make a correction directly from here? Or need I go back to my fork and make the changes there?

Could you advise the most time efficient way to make changes?

Reply:
Yes, you might press '.', this will open a web interface. There you can save and commit changes.